### PR TITLE
DBAAS-157: Set operator version dynamically in console plugin

### PR DESCRIPTION
**Pull request by yzhao583**
Set operator version dynamically in console plugin

JIra: [https://issues.redhat.com/browse/DBAAS-157](https://issues.redhat.com/browse/DBAAS-157)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 ## Local development
 
-1. `yarn build` to build the plugin, generating output to `dist` directory
-2. `yarn http-server` to start an HTTP server hosting the generated assets
+1. Add `.env` file to the root directory, add `DBAAS_OPERATOR_VERSION=OPETATOR_NAME_AND_VERSION` to the file, replace the `OPETATOR_NAME_AND_VERSION` with correct value, for example, `dbaas-operator.v0.0.1`
```
Comment (jeremyary) [inline README.md:5]: s/OPETATOR/OPERATOR in 2 places here

**Review by jeremyary — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (yzhao583) [reply README.md:5]: Typo, my bad.

Comment (yzhao583) [reply README.md]: @jeremyary Updated.

**Comment by redhatHameed** [comment]
@yzhao583 can build the docker image 'docker build -f Dockerfile -t quay.io/$USER/dbaas-dynamic-plugin .' and see if it works with you thanks I am getting error

**Comment by redhatHameed** [comment]
getting this error : any thoughts Screen Shot 2021-09-22 at 6 21 23 PM

**Comment by yzhao583** [comment]
@redhatHameed Can you try again?Thanks

**Comment by yzhao583** [comment]
Will open a new PR

**Pull request closed without merging**